torch/csrc/inductor/inductor_ops.cpp

```diff
@@ -108,8 +108,14 @@ static void resize_storage_bytes__functionalize(
     op.call(variable, new_size);
     return;
   }
+  // Don't functionalize, call the mutable op on the inner tensor.
   auto functional_impl =
       at::functionalization::impl::unsafeGetFunctionalWrapper(variable);
+  {
+    at::AutoDispatchSkipFunctionalize guard;
+    op.call(functional_impl->value(), new_size);
+    return;
+  }
```

Comment:
I suspect this change causes some existing resize tests to fail because they assume a functionalized resize

Wonder should we create a FSDP2-specific storage resize op to never-functionalize, but leave the generic `resize_storage_bytes_` op as-is?